project-charter.md

```diff
@@ -124,7 +126,7 @@ The Project should:
 
     1.  All new inbound code contributions must be accompanied by a Developer Certificate of Origin sign-off ([http://developercertificate.org)](http://developercertificate.org)
 
-    2. All contributions of code will be received and made available under the GNU General Public License, Version 2 or later, or Version 3 or later.
+    2. All contributions of code will be received and made available under the GNU General Public License, Version 2 or later, or Version 3 or later or an OSI-approved open source license defined by the software project
```

Comment:
I'm not against broadening the licensing choices, as long as projects may decide their own licensing. But maybe this specific point should be commented with the software projects thmselves, before proposing to the GB? At least, to have their input about what they think... The main problem I see with license proliferation is that maybe we're going to increase the trouble for reusing ccde between our own projects (eg, if some of them decides to go with some GPL-incompatible OSI approved license).

Reply:
+1 to getting LF feedback before making this change.

Reply:
@germonprez what do you think if we deal with this in a separate pull request, if you feel it is a change we should make, and we discuss it there, with the feedback from LF and people more involved in code?

Reply:
I'm actually for the OSI language. I think that any project the comes in would be subject to review and it is quite possible to have a different license. This does remind me that we need to have a 'contributor signoff' bot in the workflow.

Reply:
New pull request to focus on this issue: #83